docs/release-notes/v12.4.0.md

```diff
@@ -0,0 +1,42 @@
+---
+id: v12.4.0
+title: Version 12.4.0 Release Notes
+sidebar_label: v12.4.0
+image: "img/release-note-link-preview.png"
+---
+
+_April 11th, 2024_
+
+:::info
+Mia-Platform Console v12.4.0 is **now in Preview!**
+
+Console SaaS users can try out v12.4.0 latest improvements in Preview! Open a Service Request to ask for the creation of a sandbox Company in case you do not have access to any Company.
+
+For self-hosted installations, please read the [following guidelines](#how-to-update-your-console).
+:::
+
+## **Console**
+
+### Improvements
+
+### Bug Fix
+
+This version addressed the following bugs:
+
+## Fast Data
+
+### Fast Data Configurator
+
+#### Bug Fix
+
+This version addressed the following bug:
+
+* SINGLE_VIEW_NAME env var is not deleted anymore at commit when the Single View Trigger Generator is detached
+
+### Fast Data Services
+
+## Marketplace
+
```

Comment:
```suggestion

### Marketplace Updates

#### CRUD Service

The new version `7.0.1` of the CRUD service is available!

##### Improvements

* through OPEN_API_SPECIFICATION env var, now it is possible to define the OpenApi specification used for exposing Swagger
* the bulk POST and PATCH operations have been optimized to improve performances and to avoid service crashing

```